client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -202,34 +181,28 @@ const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > =
 		const domains: { name: string }[] = domain ? [ domain ] : [];
 
 		setAddingToCart( true );
-		const itemsForCart: any = getItemsForCart( domains, gSuiteProduct.product_slug, googleUsers );
-
-		shoppingCartManager
-			.addProductsToCart(
-				itemsForCart.map( ( item: any ) => fillInSingleCartItemAttributes( item, productsList ) )
-			)
-			.then( () => {
-				setAddingToCart( false );
-				const { errors } = props.cart?.messages;
-				if ( errors && errors.length ) {
-					// Stay on the page to show the relevant error(s)
-					return;
-				}
+		const cartItems: any = getItemsForCart( domains, gSuiteProduct.product_slug, googleUsers );
```

Comment:
It's really not a great idea to use `any` as a type if you can avoid it since it kind of defeats the purpose of using TypeScript. If you don't know what a function returns, you could use `unknown`, but in this case I believe `getItemsForCart` returns a type... but actually now that I look at it that function has several type errors. I've fixed them in https://github.com/Automattic/wp-calypso/pull/58881. You should be able to change this `any` to `IncompleteRequestCartProduct[]`.